src/usr/local/www/services_dyndns_edit.php

```diff
@@ -491,6 +492,9 @@ function setVisible(service) {
 				hideInput('ttl', false);
 				break;
 			case "namecheap":
+			case "cloudflare-v6":
+			case "cloudflare":
```

Comment:
Copying Comment from parent thread.

My changes are no longer needed following Cloudflare API changes.  The pfSense proxy field was a needed workaround due to how Cloudflare handled the lack of the Proxied parameter in the API call.  API changes have corrected their default behavior.

I can confirm everything works as intended with the latest pfSense release.  Only remaining changes are cleaning up unused Proxy code, and some helpful Documentation and Tips on the page.